jbmc/regression/jbmc/context-include-exclude/test_exclude_from_include.desc

```diff
@@ -0,0 +1,17 @@
+CORE
+Main.class
+--context-include Main.main --context-include 'Main.<clinit' --context-include org.cprover.MyClass --context-exclude 'org.cprover.MyClass$Inner.'
+^EXIT=10$
+^SIGNAL=0$
+.* line 10 assertion at file Main.java line 10 .*: FAILURE
+.* line 11 assertion at file Main.java line 11 .*: SUCCESS
+.* line 12 assertion at file Main.java line 12 .*: FAILURE
+WARNING: no body for function java::Main\.myMethod:\(I\)I
+WARNING: no body for function java::org\.cprover\.MyClass\$Inner\.doIt:\(I\)I
+--
+WARNING: no body for function .*clinit_wrapper
+WARNING: no body for function java::org\.cprover\.MyClass\.<init>:\(I\)V
```

Comment:
Might as well be more general -- we shouldn't see `no body for function java::org\.cprover\.MyClass\.` at all, as any exclusion should include `MyClass$Inner`

Reply:
I've listed the methods explicitly so that the difference between the expected outputs of the different test cases is clearly visible.